### PR TITLE
ci: auto-assign reviewers when PRs are raised

### PR DESCRIPTION
**Pull request by ricbha**
Closes: #158 

Signed-off-by: Richa Bharti <Richa_Bharti@mentor.com>

### CODE REVIEW

**Comment by ricbha** [comment]
@chombourger @vj-kumar Action run was successful but reviewer is not added, it may be because I don't have permission to add reviewer and assignees. I tried the same with two accounts https://github.com/BhartiRicha/github-actions-test/pull/11 it worked as expected.

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Set to true to add reviewers to pull requests
```
Comment (chombourger) [inline .github/auto_assign.yml:1]: need to add the same header as found in e.g. main.yml

File: .github/workflows/assign.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: 'Auto Assign'
```
Comment (chombourger) [inline .github/workflows/assign.yml:1]: need to add the same header as found in e.g. main.yml

**Comment by chombourger** [comment]
can we change the subject line to `ci: auto-assign reviewers when PRs are raised`

File: .github/auto_assign.yml (hunk 1 above)
Comment (ricbha) [reply .github/auto_assign.yml:12]: Done, thanks.

File: .github/workflows/assign.yml (hunk 2 above)
Comment (ricbha) [reply .github/workflows/assign.yml:12]: Done

**Comment by ricbha** [comment]
> can we change the subject line to `ci: auto-assign reviewers when PRs are raised`

Done

**Review by chombourger — APPROVED** [review]

**Review by vj-kumar — APPROVED** [review]

**Comment by chombourger** [comment]
Thanks a lot @ricbha for your contribution!

**Pull request merged**